new-backend/.env

```diff
@@ -195,7 +195,10 @@ AD_RECONNECT=true
 
 # For development and testing purposes, a user name can be supplied here.
 # Make sure to always disable this in production!
-#AD_OVERRIDE_USER_WITH_VALUE=someUser
+# AD_OVERRIDE_USER_WITH_VALUE=someUser
+# For development and testing purposes, groups can be supplied here.
+# Make sure to always disable this in production!
+# AD_OVERRIDE_USER_GROUPS_WITH_VALUE=someGroup1,someGroup2
```

Comment:
I had a quick look at the comment related to these keys and this got me thinking: should we be a bit harder on this and actually not allow these keys at all (or ignore their values in code), if `NODE_ENV=production` (see line 24). What's your opinion?

Reply:
Yes, its a really good idea actually. Prevents .env accidents even more.

Reply:
Check added:
https://github.com/hajkmap/Hajk/pull/1449/commits/9fd17fa667fc1381b488155b978b27092819ecce